modules/oe_theme_helper/src/Plugin/Field/FieldFormatter/SocialMediaLinksFormatter.php

```diff
@@ -122,7 +106,11 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
       ];
     }
 
-    return [$pattern];
+    $output = [
+      '#theme' => 'oe_theme_helper_social_media_links',
```

Comment:
Theme is used because we can't use `templates/field/field--oe-contact--oe-social-media.html.twig` - it adds top margin for oe_social_media field in Event and Project in all formatters. We need top margin for SocialMediaLinksFormatter only.